src/main/target/SPRACINGF3MINI/target.h

```diff
@@ -184,8 +184,11 @@
 #define BUTTON_A_PIN            PB1
 #define BUTTON_B_PIN            PB0
 
+#define SPEKTRUM_BIND
```

Comment:
Looks like it got removed in a merge gone wrong in https://github.com/betaflight/betaflight/commit/de3d1d527d59bba3e4468666d756c9ced389b7ce . I

This should be defiend for both SPRACINGF3MINI and TINYBEEF3, see  https://github.com/betaflight/betaflight/commit/6140a9bb573141c01cecdea86ee9b458567c8087#diff-84c688ee796941801f6d38b9a72bdaafR102